### PR TITLE
Include release date in prompts

### PR DESCRIPTION
**Pull request by elementc**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] [Searched](https://github.com/deepjyoti30/ytmdl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
_None mention 'date'_
- [x] Read [ytmdl coding conventions](https://github.com/deepjyoti30/ytmdl/blob/master/.github/CONTRIBUTING.md) and adjusted the code to meet them
_None seemed applicable_
- [x] Checked the code with [pylama](https://github.com/klen/pylama)
_Plenty of hits, but none in song.py_
- [x] Made the pull request to the **unstable** branch
_Sure._

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New feature

---

### Description of your *pull request* and other information

In short, if a song has been released on multiple albums by the same artist, the default prompt for metadata is very unhelpful:
image

This adds the release date to that prompt, helping to disambiguate which one we're grabbing:
image

To be clear, it would probably be better to include a "default" mode where we use the youtube music metadata here, but, I mean, this change unblocks me.

### CODE REVIEW

**Comment by deepjyoti30** [comment]
@elementc Thanks for the PR but adding release date to the results might not be the best option. I understand the concern that if the same song is available from various platforms, it will have different dates but what would be more helpful and visually pleasing to the user is conditional dates showing.

What we can do is if there are more than 3 results with the exact same song name, we can show the dates for all 3 of them. I feel like this can be a better option than showing it for all of them.
